### PR TITLE
tidb-binlog: update docs about checkpoint & initial-commit-ts

### PR DESCRIPTION
**Pull request by july2993**
### What is changed, added or deleted? 

Update docs about checkpoint & initial-commit-ts

### What is the related PR or file link(s)? 

dev/v3.0

### CODE REVIEW

File: dev/reference/tidb-binlog/deploy.md [hunk 1]
```diff
@@ -143,24 +143,8 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
...
-
-    ```
-    INFO[0000] [pd] create pd client with endpoints [http://192.168.199.118:32379]
-    INFO[0000] [pd] leader switches to: http://192.168.199.118:32379, previous:
-    INFO[0000] [pd] init cluster id 6569368151110378289
-    2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
-    ```
-
-    该命令会输出 `meta: &{CommitTS:400962745252184065}`，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
+    如果从最近的时间点开始同步，inital_commit_ts 使用 -1 即可。
```
Comment (ericsyh) [inline dev/reference/tidb-binlog/deploy.md]: 我们默认值就是 -1，ansible 只要里只要不填写 `inital_commit_ts` 就可以了吧？

Comment (july2993) [reply dev/reference/tidb-binlog/deploy.md]: 不填就是默认值，填了就填什么就什么

**Comment by TomShawn** [comment]
@july2993 When the content is ready, please also update v3.1 as well, thanks~

**Comment by july2993** [comment]
@TomShawn @ericsyh PTAL

File: dev/reference/tidb-binlog/deploy.md (hunk 1 above)
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]: 将 inital_commit_ts 的值设为 `-1` 即可？
```suggestion

    如果从最近的时间点开始同步，inital_commit_ts 使用 `-1` 即可。
```

File: dev/reference/tidb-binlog/deploy.md [hunk 2]
```diff
@@ -143,24 +143,8 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
...
+    如果下游为 MySQL/TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
    
    如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
```

File: dev/reference/tidb-binlog/deploy.md [hunk 3]
```diff
@@ -446,8 +430,9 @@ Drainer="192.168.0.13"
         -ignore-schemas string
             db 过滤列表 (默认 "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql,test")，
             不支持对 ignore schemas 的 table 进行 rename DDL 操作
-        -initial-commit-ts (默认为 0)
+        -initial-commit-ts (默认为 -1)
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
        -initial-commit-ts（默认为 `-1`）
```

File: dev/reference/tidb-binlog/deploy.md [hunk 4]
```diff
@@ -446,8 +430,9 @@ Drainer="192.168.0.13"
...
             如果 Drainer 没有相关的断点信息，可以通过该项来设置相关的断点信息
+            -1 drainer 会自动从 PD 获取一个最新的时间戳。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
            该参数值为 `-1` 时，Drainer 会自动从 PD 获取一个最新的时间戳。
```

File: dev/reference/tidb-binlog/deploy.md [hunk 5]
```diff
@@ -559,8 +544,18 @@ Drainer="192.168.0.13"
         port = 3306
 
         [syncer.to.checkpoint]
-        # 当下游是 MySQL 或 TiDB 时可以开启该选项，以改变保存 checkpoint 的数据库
+        # 当 checkpoint type 是 mysql 或 tidb 时可以开启该选项，以改变保存 checkpoint 的数据库
         # schema = "tidb_binlog"
+        # 目前只支持 mysql 或者 tidb 类型，你可以去掉注释来控制 checkpoint 保存到哪里。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
        # 目前只支持 mysql 或者 tidb 类型。可以去掉注释来控制 checkpoint 保存的位置。
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 6]
```diff
@@ -144,6 +144,12 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
 
 1. 获取 initial_commit_ts
 
+    如果从最近的时间点开始同步，inital_commit_ts v3.0.6 开始版本使用 -1 即可, 否则参考后面方法获取一个最新的时间戳。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
    如果从最近的时间点开始同步，从 v3.0.6 开始 inital_commit_ts 使用 `-1` 即可，否则参考下文方法获取一个最新的时间戳。
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 7]
```diff
@@ -144,6 +144,12 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
...
+
+    如果下游为 MySQL/TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
    如果下游为 MySQL 或 TiDB，为了保证数据的完整性，需要进行全量数据的备份与恢复，必需使用全量备份的时间戳。
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 8]
```diff
@@ -144,6 +144,12 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
...
+
+    如何获取一个最新的时间戳:
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
    获取一个最新时间戳的方法：
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 9]
```diff
@@ -445,8 +451,9 @@ Drainer="192.168.0.13"
         -ignore-schemas string
             db 过滤列表 (默认 "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql,test")，
             不支持对 ignore schemas 的 table 进行 rename DDL 操作
-        -initial-commit-ts (默认为 0)
+        -initial-commit-ts (默认为 0, v3.0.6 开始默认值为 -1)
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
        -initial-commit-ts（默认为 0，v3.0.6 开始默认值为 -1）
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 10]
```diff
@@ -445,8 +451,9 @@ Drainer="192.168.0.13"
...
             如果 Drainer 没有相关的断点信息，可以通过该项来设置相关的断点信息
+            -1 drainer 会自动从 PD 获取一个最新的时间戳。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
            该参数值为 -1 时，Drainer 会自动从 PD 获取一个最新的时间戳。
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 11]
```diff
@@ -558,8 +565,19 @@ Drainer="192.168.0.13"
         port = 3306
 
         [syncer.to.checkpoint]
-        # 当下游是 MySQL 或 TiDB 时可以开启该选项，以改变保存 checkpoint 的数据库
+        # 当 checkpoint type 是 mysql 或 tidb 时可以开启该选项，以改变保存 checkpoint 的数据库
         # schema = "tidb_binlog"
+        #
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 12]
```diff
@@ -558,8 +565,19 @@ Drainer="192.168.0.13"
...
+        # v3.0.6 开始支持配置 checkpoint 的保存类型。
+        # 目前只支持 mysql 或者 tidb 类型，你可以去掉注释来控制 checkpoint 保存到哪里。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
        # 目前只支持 mysql 或者 tidb 类型。可以去掉注释来控制 checkpoint 保存的位置。
```

File: v3.1/reference/tidb-binlog/deploy.md [hunk 13]
```diff
@@ -142,24 +142,8 @@ Pump 和 Drainer 均可部署和运行在 Intel x86-64 架构的 64 位通用硬
...
-
-    ```
-    INFO[0000] [pd] create pd client with endpoints [http://192.168.199.118:32379]
-    INFO[0000] [pd] leader switches to: http://192.168.199.118:32379, previous:
-    INFO[0000] [pd] init cluster id 6569368151110378289
-    2018/06/21 11:24:47 meta.go:117: [info] meta: &{CommitTS:400962745252184065}
-    ```
-
-    该命令会输出 `meta: &{CommitTS:400962745252184065}`，CommitTS 的值作为 Drainer 初次启动使用的 `initial-commit-ts` 参数的值。
+    如果从最近的时间点开始同步，inital_commit_ts 使用 -1 即可。
```
Comment (TomShawn) [inline v3.1/reference/tidb-binlog/deploy.md]: see the comment for line 146 in `dev` version. 
```suggestion
    
    如果从最近的时间点开始同步，inital_commit_ts 使用 -1 即可。
```

**Comment by july2993** [comment]
@TomShawn PTAL

File: dev/reference/tidb-binlog/deploy.md [hunk 14]
```diff
@@ -446,8 +432,9 @@ Drainer="192.168.0.13"
         -ignore-schemas string
             db 过滤列表 (默认 "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql,test")，
             不支持对 ignore schemas 的 table 进行 rename DDL 操作
-        -initial-commit-ts (默认为 0)
+        -initial-commit-ts（默认为 `-1`）
             如果 Drainer 没有相关的断点信息，可以通过该项来设置相关的断点信息
+            该参数值为 `-1` 时，Drainer 会自动从 PD 获取一个最新的时间戳。
```
Comment (TomShawn) [inline dev/reference/tidb-binlog/deploy.md]:
```suggestion
            该参数值为 `-1` 时，Drainer 会自动从 PD 获取一个最新的时间戳
```

File: v3.0/reference/tidb-binlog/deploy.md [hunk 15]
```diff
@@ -445,8 +451,9 @@ Drainer="192.168.0.13"
         -ignore-schemas string
             db 过滤列表 (默认 "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql,test")，
             不支持对 ignore schemas 的 table 进行 rename DDL 操作
-        -initial-commit-ts (默认为 0)
+        -initial-commit-ts（默认为 `0`，v3.0.6 开始默认值为 `-1`）
             如果 Drainer 没有相关的断点信息，可以通过该项来设置相关的断点信息
+            该参数值为 -1 时，Drainer 会自动从 PD 获取一个最新的时间戳。
```
Comment (TomShawn) [inline v3.0/reference/tidb-binlog/deploy.md]:
```suggestion
            该参数值为 -1 时，Drainer 会自动从 PD 获取一个最新的时间戳
```

File: v3.1/reference/tidb-binlog/deploy.md [hunk 16]
```diff
@@ -444,8 +430,9 @@ Drainer="192.168.0.13"
         -ignore-schemas string
             db 过滤列表 (默认 "INFORMATION_SCHEMA,PERFORMANCE_SCHEMA,mysql,test")，
             不支持对 ignore schemas 的 table 进行 rename DDL 操作
-        -initial-commit-ts (默认为 0)
+        -initial-commit-ts (默认为 `-1`)
```
Comment (TomShawn) [inline v3.1/reference/tidb-binlog/deploy.md]:
```suggestion
        -initial-commit-ts（默认为 `-1`）
```

File: v3.1/reference/tidb-binlog/deploy.md [hunk 17]
```diff
@@ -444,8 +430,9 @@ Drainer="192.168.0.13"
...
             如果 Drainer 没有相关的断点信息，可以通过该项来设置相关的断点信息
+            该参数值为 `-1` 时，Drainer 会自动从 PD 获取一个最新的时间戳。
```
Comment (TomShawn) [inline v3.1/reference/tidb-binlog/deploy.md]:
```suggestion
            该参数值为 `-1` 时，Drainer 会自动从 PD 获取一个最新的时间戳
```

**Pull request merged**